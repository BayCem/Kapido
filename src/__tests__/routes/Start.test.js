import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Start from '../../routes/Start';

configure({ adapter: new Adapter() });
it('renders coorectly', () => {
  const wrapper = shallow(<Start />);
  expect(wrapper).toMatchSnapshot();
});
