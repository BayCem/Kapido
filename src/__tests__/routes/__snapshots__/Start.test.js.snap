// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders coorectly 1`] = `
<h1>
  Start
</h1>
`;
